wails-frontend/src/pages/success.tsx

```diff
@@ -21,10 +22,16 @@ const Success = () => {
   };
 
   return (
-    <section className="Success">
-      <div>Success !!</div>
-      <div>{successMsg(req)}</div>
-    </section>
+    <div className="bg-primary flex flex-col justify-center items-center h-screen w-full">
+      <div className="w-1/4  max-w-xs  min-w-fit flex flex-col justify-center items-center">
```

Comment:
**max-w-xs** I am not sure we need it!

Reply:
We talked about it in the pr [click here](https://github.com/massalabs/thyra-plugin-wallet/pull/207#discussion_r1196797828)

It's mainly for esthetic purposes when the user stretches the screen